programs/keeper/Keeper.cpp

```diff
@@ -224,12 +224,12 @@ struct Keeper::KeeperHTTPContext : public IHTTPContext
 
     uint64_t getMaxFieldNameSize() const override
     {
-        return context->getConfigRef().getUInt64("keeper_server.http_max_field_name_size", 1048576);
+        return context->getConfigRef().getUInt64("keeper_server.http_max_field_name_size", 131072);
```

Comment:
I'd prefer to see something like `128 * 1024` because it's hard to understand this number

Reply:
Done